index.html

```diff
@@ -1949,56 +1949,573 @@ <h2>
+            </li>
+            <li>If <var>details</var>["<a>postalCode</a>"] is present, <a>strip
+            leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>postalCode</a>"] and set
+            <var>address</var>.<a>[[\postalCode]]</a> to the result.
+            </li>
+            <li>If <var>details</var>["<a>sortingCode</a>"] is present,
+            <a>strip leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>sortingCode</a>"] and set
+            <var>address</var>.<a>[[\sortingCode]]</a> to the result.
+            </li>
+            <li>If <var>details</var>["<a>sortingCode</a>"] is present,
+            <a>strip leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>sortingCode</a>"] and set
+            <var>address</var>.<a>[[\sortingCode]]</a> to the result.
+            </li>
+            <li>If <var>details</var>["<a>organization</a>"] is present,
+            <a>strip leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>organization</a>"] and set
+            <var>address</var>.<a>[[\organization]]</a> to the result.
+            </li>
+            <li>If <var>details</var>["<a>recipient</a>"] is present, <a>strip
+            leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>recipient</a>"] and set
+            <var>address</var>.<a>[[\recipient]]</a> to the result.
+            </li>
+            <li>If <var>details</var>["<a>phone</a>"] is present, <a>strip
+            leading and trailing ASCII whitespace</a> from
+            <var>details</var>["<a>phone</a>"] and set
+            <var>address</var>.<a>[[\phone]]</a> to the result.
+            </li>
+            <li>Return <var>address</var>.
+            </li>
+          </ol>
+        </section>
+        <section>
+          <h2>
+            <dfn>toJSON()</dfn> method
+          </h2>
+          <p>
+            When called, runs [[!WEBIDL]]'s <a data-cite=
+            "WEBIDL#default-tojson-operation">default toJSON operation</a>.
+          </p>
+        </section>
+        <section>
+          <h2>
+            <dfn>country</dfn> attribute
+          </h2>
+          <p>
+            When getting, returns the value of the <a>PaymentAddress</a>'s
+            <a>[[\country]]</a> internal slot.
+          </p>
+        </section>
+        <section>
+          <h2>
+            <dfn>addressLine</dfn> attribute
+          </h2>
+          <p>
+            Represents the <a>address line</a> of the address. When getting,
+            returns the value of the <a>PaymentAddress</a>'s
+            <a>[[\addressLine]]</a> internal slot.
+          </p>
+        </section>
+        <section>
+          <h2>
+            <dfn>region</dfn> attribute
+          </h2>
+          <p>
+            Represents the <a>region</a> of the address. When getting, returns
```

Comment:
Links to the wrong place, here and in several other "represents". You can tell because in the preview they are in code font, but they should not be.